### ISSUE TITLE
Container won't save after changing the port in settings.

### ISSUE THREAD

**Issue by lawrencebr** [docker/kitematic#2305]
### Expected behavior
Container saves new port assignment.

### Actual behavior
Error is thrown:
(HTTP code 409) unexpected - Conflict. The container name "/mongo" is already in use by container fc70066d6f1699305954d937ebf8576edf3fd54a166eb468f92bf7ebec3b805e. You have to remove (or rename) that container to be able to reuse that name.

### Information about the Issue
I am using windows containers, not linux containers.
The container must be corrupted after that because I can't get it to start again, and have to remove it.
running: docker ps -a shows this container has a status of dead after this problem.

Versions used:
Windows 10, Docker 1.13.0 (9795), Kitematic v0.35.4

### Steps to reproduce the behavior

  1. Create Mongo container
  2. Stop container
  3. Edit port assignment in container setting in Kitematic
  4. Save

**Comment by FrenchBen** [comment]
Couple of things:
1. Using containers on Windows usually still means Linux Containers, unless you're on Windows Server 2016
2. I'm unable to replicate this issue. When I add the port and click on 'Save' it starts up and I get the following logs: 
```
2017-01-27T19:37:03.756+0000 I COMMAND  [initandlisten] setting featureCompatibilityVersion to 3.4
2017-01-27T19:37:03.756+0000 I NETWORK  [thread1] waiting for connections on port 27017
```

There's something else going on with your system.
Can you provide the following:
* Kitematic version
* Output of `docker version`
* Full output of `docker system info`

**Comment by lawrencebr** [comment]
No, I right clicked on Docker and switched to windows containers.  In addition I used mongo:windowsservercore

Sorry I didn’t specify that before.

Brad Lawrence
BYU-Idaho
Software Engineer
Kimball 250A
lawrencebr@byui.edu<mailto:lawrencebr@byui.edu>
(208) 496-7233

**Comment by FrenchBen** [comment]
Thanks for adding some crucial pieces for us to debug this - I'll try to replicate this using Windows Containers.
